### PR TITLE
Feature#1/ui login

### LINKED ISSUE #1: Login UI Design
# REQUIREMENTS
- Develope the Ui Design for the login screen.
- Add folder structure for he project.

# DESIGN
Captura de Pantalla 2021-03-08 a la(s) 10 19 46

# Acceptance Criteria
List all required acceptance criteria
- [ ] Covid modification to design.
- [ ] UI as close as posible.

### PR DESCRIPTION
**Pull request by Joaquin-Niembro**
# Description
What did you implemented/Why did you implemented this:
-Implemeneted the UI design and coded it.
-Form validation using Formik.
 
 # Testing
-Git clone repo
-git branch Feature#1uilogin
-npm install
-npm run start
-npm run android
 
  ## Screenshots 
Frame 1

### CODE REVIEW

File: AlertaCovid/src/styles/WelcomeStyles/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        height: 53,
+        width: 222,
+        bottom: 950,
+        left: 100,
+        alignItems: 'center',
+        justifyContent: 'center',
+        borderRadius: 13,
+        padding: 20
+    }
+})
```
Comment (LeyaimMagma) [inline AlertaCovid/src/styles/WelcomeStyles/index.js]: Always a blank space at the end

File: AlertaCovid/src/styles/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export * from './WelcomeStyles'
+export * from './colors'
+export * from './SignIn'
```
Comment (LeyaimMagma) [inline AlertaCovid/src/styles/index.js]: Always a blank space at the end

File: AlertaCovid/src/utils/constants/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export const constants = {
+    WelcomeTitle: 'WELCOME TO',
+    WelcomeTitle2: 'COVID-ALERT',
+    Started: 'GET STARTED',
+    SignIn: 'Sign In',
+    ForgotPassword: 'Forgot password?'
+}
```
Comment (LeyaimMagma) [inline AlertaCovid/src/utils/constants/index.js]: Always a blank space at the end

File: AlertaCovid/src/utils/index.js [hunk 4]
```diff
@@ -0,0 +1 @@
+export * from './constants'
```
Comment (LeyaimMagma) [inline AlertaCovid/src/utils/index.js]: Always a blank space at the end

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Review by fernandoamz — CHANGES_REQUESTED** [review]
some files have the indentation with 4 spaces and should be 2

**Review by ChantalRamirez — DISMISSED** [review]
Excelent design

**Review by LeyaimMagma — APPROVED** [review]

**Review by fernandoamz — APPROVED** [review]

**Review by ChantalRamirez — APPROVED** [review]

**Pull request merged**